src/main/kotlin/graphics/scenery/Icosphere.kt

```diff
@@ -124,7 +133,7 @@ open class Icosphere(val radius: Float, val subdivisions: Int) : Mesh("Icosphere
         val i = this.addVertex(middle)
 
         // store it, return index
-        middlePointIndexCache.put(key, i)
+        middlePointIndexCache[key] = i
```

Comment:
@elect86 Do you notice these manually, or lean on an IDE cleanup/simplification action? If so, what is it?

Reply:
If suggested, "Alt+Enter" opens the action dialog and then you select the suggestion